src/App.tsx

```diff
@@ -1,10 +1,62 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+enum TypeFilterin {
+  All = 'all',
+  Completed = 'comleted',
+  Active = 'active',
+}
+
+const USER_ID = 9940;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [typeOfFiltering, setTypeOfFiltering] = useState('all');
+  const [dataError, setError] = useState<Error | null>(null);
+  const [closeErrorButton, setCloseErrorButton] = useState(false);
+  const [hideError, setHideError] = useState(false);
```

Comment:
no need for this variable. you can check if the error message should be shown by help of `dataError` variable